File: RepoTransBench/target_projects/Java/C#/czz1233_GBByPass/public_tests/RandomUtilBranchCoveragePublicTest.cs
using Xunit;
using GBByPass;

namespace PublicTests
{
    public class RandomUtilBranchCoveragePublicTest
    {
        [Fact]
        public void TestRandomString_lengthTwo()
        {
            string s = RandomUtil.RandomString(2);
            Assert.NotNull(s);
            Assert.Equal(2, s.Length);
            Assert.Matches("^[A-Za-z0-9]{2}$", s);
        }

        [Fact]
        public void TestRandomString_mediumLength()
        {
            string s = RandomUtil.RandomString(50);
            Assert.NotNull(s);
            Assert.Equal(50, s.Length);
            Assert.Matches("^[A-Za-z0-9]{50}$", s);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/czz1233_GBByPass/public_tests/random_util_branch_coverage_public_test.go
package public_tests

import (
	"testing"
	"regexp"

	"github.com/stretchr/testify/assert"
	"project/company"
)

func TestRandomString_LengthTwo_Public(t *testing.T) {
	s := company.RandomString(2)
	assert.NotNil(t, s)
	assert.Equal(t, 2, len(s))
	matched, _ := regexp.MatchString("^[A-Za-z0-9]{2}$", s)
	assert.True(t, matched)
}

func TestRandomString_MediumLength_Public(t *testing.T) {
	s := company.RandomString(50)
	assert.NotNil(t, s)
	assert.Equal(t, 50, len(s))
	matched, _ := regexp.MatchString("^[A-Za-z0-9]{50}$", s)
	assert.True(t, matched)
}